airflow/providers/microsoft/azure/example_dags/example_sftp_to_wasb.py

```diff
@@ -69,4 +69,4 @@ def delete_sftp_file():
         blob_name=BLOB_PREFIX + SAMPLE_FILENAME,
     )
 
-    transfer_files_to_sftp_step >> transfer_files_to_azure >> delete_blob_file_step >> delete_sftp_file()
+    transfer_files_to_sftp_step >> transfer_files_to_azure >> delete_blob_file_step >> delete_sftp_file
```

Comment:
I think this one is a mistake and it needs the `()`

Reply:
Ah yeah. It was by accident. And it should be handled by #20933  I guess.